.github/workflows/ci.yml

```diff
@@ -71,7 +71,12 @@ jobs:
 
       - name: Run unittest tests with coverage
         run: |
+          echo "OPENAI_API_KEY=${{ secrets.OPENAI_API_KEY }}" > .env
```

Comment:
it should just read it from the environment if injected, shouldn’t need to write it to the .env